lib/tasks/autotest.rake

```diff
@@ -80,6 +80,7 @@ class AutotestSetup
           repository_folder: @assg_short_id,
           enable_test: true,
           token_start_date: Time.current,
+          token_end_date: Time.current,
```

Comment:
This is for seeding, but it should also be okay to set this to `nil` so that the field is blank (this is the default behaviour)